roles/nova-data/tasks/main.yml

```diff
@@ -27,6 +27,17 @@
 
 - modprobe: name=nbd state=present
 
+- name: Turning off SMT mode for ppc64le
+  command: "ppc64_cpu --smt=off"
+  when: ansible_architecture == "ppc64le"
```

Comment:
Is there strong reason to always do this in a openstack environment? IF not, this should probably be a variable to disable it or not, and default it to being disabled.